hal-base/hal/functions.py

```diff
@@ -494,14 +484,14 @@ def cleanNotifier(notifier):
 #############################################################################
 # PDP
 #############################################################################
-getPDPTemperature = _STATUSFUNC("getPDPTemperature", C.c_double)
```

Comment:
Don't you need to fix the callers of these functions in both the java and c++ wpilib? Not just the HAL api?